### DISCUSSION TITLE
Cannot connect to ETOP-HT-01AC Thermostat

### DISCUSSION THREAD

**Discussion by robex2005** [make-all/tuya-local#648] [Q&A]
Hello, 

Trying to connect to a ETOP-HT-01AC Thermostat, with no luck
Actually, the thermostat is connected to a E-GW-R gateway, with another  ETOP-HT-01AC (so it-s a gateway with 2 thermostats for 2 zones)
With the office Tuya app, the device and the 2 thermostats are showing , and i can control them...
With the local-tuya app, i have a message _**Unable to connect to your device with those details. It could be an intermittent issue, or they may be incorrect.**_
I have local key for the gateway and i have a device id for gateway and 2 id-s for the 2 thermostats
I-ve tried connecting with the thermostat id and gateway key, no luck. I-ve tried connecting the gateway, with his device id and key, still no luck

I-ve managed to get some progress with the other localtuya integration, it seems that is connecting to the device, but the dp-s are not automatically populated
If i set them manually, i get -1 values for the different dp-s
I-ve collected the dp-s form the tuya iot console, so it must be right...

Any ideas?

**Comment by make-all** [comment]
Gateway support will be in the next release.
